config/settings.yml

```diff
@@ -3,3 +3,4 @@ links:
     home_page: https://www.railstutorial.org/
     help_page: http://www.railstutorial.org/help
     book_page: https://www.railstutorial.org/book
+    
```

Comment:
remove empty line
and follow number 12 https://docs.google.com/document/d/104Csp4-vamVos5DEbi372nLBEfmqhj7h1ENYO8ebjCo/edit